{{cookiecutter.project_slug}}/root/docker-compose.yml

```diff
@@ -5,16 +5,14 @@ services:
     image: postgres:14
 
     environment:
-      POSTGRES_USER: postgres
-      POSTGRES_PASSWORD: postgres
-      # Password removed
-      POSTGRES_DB: relecloud
-      POSTGRES_HOST: db
-      POSTGRES_PORT: 5432
+      DBSERVER_USER: postgres
+      DBSERVER_DB: relecloud
+      DBSERVER_HOST: db
+      DBSERVER_PORT: 5432
 
     ports:
       - 5432:5432
-      - {{cookiecutter.web_port}}:{{cookiecutter.web_port}}}
```

Comment:
I was looking at this in mine and it confused me, shouldnt that port be exposed on the other service, not the db service?